### DISCUSSION TITLE
将配置文件一起放入到生成目录

### DISCUSSION THREAD

**Discussion by lamouse** [xmake-io/xmake#3456] [Q&A]
我项目有一些配置文件，需要从src放到项目的生成目录中，除了写命令还没有一些简单的办法类似cmake的file(COPY ${TEMPLATE_FILES} DESTINATION jsp-tem)

**Comment by star-hengxing** [comment]
```lua
function copy(file) 
    after_build(function (target)
        os.cp(file, target:targetdir())
    end)
end

target("main")
    copy("xxx")
```

**Reply by lamouse** [reply]
这个太简单了，我匹配好多个文件和目录，环境不一样匹配的文件不一样，不过还是感谢

**Reply by waruqi** [reply]
add_configfiles/set_configdir

https://xmake.io/#/zh-cn/manual/project_target?id=targetadd_configfiles
